### PR TITLE
APP-570 Fix the search field disappears when focused

### PR DESCRIPTION
**Pull request by iosshervin**
- Implemented the search bar on navigation bar. So, the search bar is moved from table view header to navigation bar.

### CODE REVIEW

File: app/Jasonette/JasonViewController.m [hunk 1]
```diff
@@ -1302,8 +1298,9 @@ - (void)setupHeader: (NSDictionary *)body{
                         self.searchController.searchBar.returnKeyType = UIReturnKeySearch;
                         self.searchController.searchBar.barTintColor = backgroundColor;
                         self.searchController.searchBar.tintColor = color;
-                        self.searchController.searchBar.backgroundColor = backgroundColor;
+                        //self.searchController.searchBar.backgroundColor = backgroundColor;
```
Comment (rsimenok) [inline app/Jasonette/JasonViewController.m]: Let's remove this line if it's not needed anymore

File: app/Jasonette/JasonViewController.m [hunk 2]
```diff
@@ -1321,8 +1318,9 @@ - (void)setupHeader: (NSDictionary *)body{
                             }
                         }
 
-                        if(!self.tableView.tableHeaderView) {
-                            self.tableView.tableHeaderView = self.searchController.searchBar;
+                        if (@available(iOS 13.0, *)) {
```
Comment (rsimenok) [inline app/Jasonette/JasonViewController.m]: What is the lowest iOS version for this project?
In my project file it is iOS 13, if so, do we need this `if` statement?

**Review by rsimenok — APPROVED** [review]

File: app/Jasonette/JasonViewController.m (hunk 2 above)
Comment (iosshervin) [reply app/Jasonette/JasonViewController.m]: no, i will remove the **if** statement.

File: app/Jasonette/JasonViewController.m (hunk 1 above)
Comment (iosshervin) [reply app/Jasonette/JasonViewController.m]: sure.

**Pull request merged**